index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://code.jquery.com/jquery-3.6.4.min.js" defer></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="homepage">
```

Comment:
- [ ] I can see some overflows in your design, kindly note that for good user experience, we try as much as possible to ensure that our website Is responsive and a user doesn't have to scroll to view contents.

things that will help are: 
1. use percentages % for images and background
2. use rem and em for  your texts
3. use flexbox or grid
![image](https://github.com/denigit5/Portfolio/assets/63473584/ec4d8aff-317b-41c8-a126-462a32ffee62)